src/kvstore/Part.cpp

```diff
@@ -441,6 +423,23 @@ bool Part::preProcessLog(LogID logId,
     return true;
 }
 
+ResultCode Part::toResultCode(raftex::AppendLogResult res) {
+    switch (res) {
+        case raftex::AppendLogResult::SUCCEEDED:
+            return ResultCode::SUCCEEDED;
+        case raftex::AppendLogResult::E_NOT_A_LEADER:
+            return ResultCode::ERR_LEADER_CHANGED;
+        case raftex::AppendLogResult::E_WRITE_BLOCKING:
+            return ResultCode::ERR_WRITE_BLOCK_ERROR;
+        default:
```

Comment:
Maybe we can assign AppendLogResult::E_BUFFER_OVERFLOW to another ResultCode?

Reply:
There are already too many codes inside ResultCode and cpp2::ErrorCode. 
We could check the internal consensus error code with this log.